tests/automation.js

```diff
@@ -21,6 +21,7 @@ casper.options.onWaitTimeout = function() {
 var gfxTests = [
   { name: "gfx/AlertTest", maxDifferent: 1942 },
   { name: "gfx/CanvasTest", maxDifferent: 0 },
+  { name: "gfx/CanvasWithHeaderTest", maxDifferent: 2000 },
```

Comment:
I reduced maxDifferent to 1400 to be compatible with that on Mac. It's value varies a lot with different platform. On my ubuntu desktop the value is 0.